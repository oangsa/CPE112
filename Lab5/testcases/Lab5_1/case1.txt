Peek
Dequeue
Enqueue 1
Enqueue 2
Enqueue 3
Peek
Dequeue
Peek
END
