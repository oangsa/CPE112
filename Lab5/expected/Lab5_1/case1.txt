Queue is empty
Queue is empty
1 
1 
2 
2 3 
